tests/SpecTests/TransactionsSpecTest.php

```diff
@@ -56,89 +56,62 @@ public function tearDown()
         parent::tearDown();
     }
 
-    /**
-     * Kill all sessions on the cluster.
-     *
-     * This will clean up any open transactions that may remain from a
-     * previously failed test. For sharded clusters, this command will be run
-     * on all mongos nodes.
-     */
-    private static function killAllSessions()
```

Comment:
This was moved to the bottom of the file, to be consistent with ordering of public, protected, and private members.